contracts/stork_chainlink_adapter/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@storknetwork/stork_chainlink_adapter",
```

Comment:
It might make sense to have a single npm package called `@storknetwork/sdk` or something.